client/src/components/atoms/Form/Input.module.scss

```diff
@@ -65,8 +63,8 @@
         font-size: $font-size-base;
         color: $brand-grey-light;
         font-weight: $font-weight-base;
-        transition: .2s ease-out;
-        opacity: .7;
+        transition: 0.2s ease-out;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)